tests/test_repository_local.py

```diff
@@ -1,63 +1,64 @@
-    "sha256:5e1368d13f1774852f9e435260be19ad726bbfb501b80472f61c2dc768a0692a",
-    "sha256:60881c79eb72cb75bd0a4be5e31c9e431739146c4184a2618cabea3938418984",
-    "sha256:6120b62a642a40e47eb6c9ff00c02be69158fc7f7c5ff78e42a2c739d1c57cd6",
-    "sha256:65c223e77f87cb463191ace3398e0a6d84ce4ac575d42eb412a220b099f593d6",
-    "sha256:6fbf8db55710959344502b58ab937424173ad8b5eb514610bcf56b119caa350a",
-    "sha256:74aadea668c94eef4ceb09be3d0eae6619e28b4f1ced4e29cd43a05bb2cfd7a4",
-    "sha256:7be1efa623e1ed91b15b1e62e04c536def1d75785eb930a0b8179ca6b65ed16d",
-    "sha256:83266cdede210393889471b0c2631e78da9d4692fcca875af7e958ad39b897ee",
-    "sha256:86c68a3f8246495962446c6f96f6a27f182b91208187b68f1e87ec3dfd29fa32",
-    "sha256:9163f7743cf9991edaddf9cf886708e288fab38e1b9fec9c41c15c85c8f7f147",
-    "sha256:97d9f338f91b7927893ea6500b953e4b4b7e47c6272222992bb76221e17056ff",
-    "sha256:a7930e73a4359b52323d09de6d6860840314aa09346cbcf4def8875e1b07ebc7",
-    "sha256:ada8a42c493e4934a1a8875c2bc9efcb1b88c09883f70375bfa053ab32d6a118",
-    "sha256:b0bc2d83cc0ba0e8f0d9eca2ffe07f72f33bec7d84547071e7e875d4cca8272d",
-    "sha256:b5412a65605c642adf3e1544b59b8537daf5696dedadd2b3cbebc42e24da45ed",
-    "sha256:ba6b5205fced1625b6d9d55f9ef422f9667c5d95f18f07c0611eb964a3355331",
-    "sha256:bcaf3d86385daaab0ae51c9c53ebe70a6c1c5dfcb9e311b13517e04773ddf6b6",
-    "sha256:cfa15570ecec1ea6bee089e86fd4deae6208c96a811344ce246de5e5c9ac824a",
-    "sha256:d3e3063af1fa6b59e255da9a812891cdaf24b90fbaf653c02797871069b7c4c9",
-    "sha256:d9cfe26ecea2fec320cd0cac400c9c2435328994d23596ee6df63945fe7292b0",
-    "sha256:e5ef800ef8ef9ee05ae9a5b7d7d9cf7d6c936b32e312e54823faca3034ee16ab",
-    "sha256:f1366150acf611d09d37ffefb3559ed3ffeb1713643d3cd10716d6c5da3f83fb",
-    "sha256:f4eb9747a37120b35f59c8e96265e87b0c432ff010d32fc0772992aa14659502",
-    "sha256:f8264463cc08cd696ad17e4bf3c80f3344628c04c11ffdc545ddf0798bc17316",
-    "sha256:f8ba54848dfe280b1be0d6e699544cee4ba10d566f92464538063d9e645aed3e",
-    "sha256:f93d1edcaea7b6a7a8fbf936f4492a9a0ee0b4cb281efebd5e1dd73e5e432c71",
-    "sha256:fc8865c7e0ac25ddd71036c2b9a799418b32d9acb40400d345b8791b6e1058cb",
-    "sha256:fce6b0cb9ade1546178c031393633b09c4793834176496c99a94de0bfa471b27",
-    "sha256:fde17c52d7ce7d55a9fb263b57ccb5da6439915b5c7105617eb21f636bb1bd9c",
-}
 
+def test_get_hashes_local_repository_cache_miss(pip_conf, from_line, pypi_repository):
+    EXPECTED = {
+        "sha256:71403033c0545516cc5066c9196d9490affae65a865af3198438be6923e4762e"
+    }
 
-@pytest.mark.network
-def test_get_hashes_local_repository_cache_miss(from_line, pypi_repository):
     existing_pins = {}
     local_repository = LocalRequirementsRepository(existing_pins, pypi_repository)
     with local_repository.allow_all_wheels():
-        hashes = local_repository.get_hashes(from_line("cffi==1.9.1"))
+        hashes = local_repository.get_hashes(from_line("small-fake-with-deps==0.1"))
```

Comment:
Could we test use `small-fake-a==0.1` instead of `small-fake-with-deps==0.1`? The package `small-fake-with-deps` useful only if we have to test its dependencies.